src/components/OrderPage/index.js

```diff
@@ -1,9 +1,52 @@
 import {useEffect, useState} from "react"
 import ItemCard from "../ItemCard"
 import './OrderPage.scss'
+import OrderCard from "./OrderCard";
+import userEvent from "@testing-library/user-event";
 
 const OrderPage = ({activeRestaurantId}) => {
     const [foodItemData, setItemData] = useState([])
+    const [orderObject, setOrderObject] = useState([])
```

Comment:
This should be renamed since it's an array, simply changing it to `orderArray` or `orderItems` instead does the job